pom.xml

```diff
@@ -1,193 +1,226 @@
-          </configuration>
-          <reportSets>
-              <reportSet>
-                  <id>default</id>
-                  <reports>
-                      <report>javadoc</report>
-                  </reports>
-              </reportSet>
-          </reportSets>
-        </plugin>
-
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-jxr-plugin</artifactId>
-          <configuration>
-              <linkJavadoc>true</linkJavadoc>
-          </configuration>
-          <version>2.5</version>
-        </plugin>
-
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-checkstyle-plugin</artifactId>
-          <version>2.17</version>
-          <configuration>
-              <configLocation>${basedir}/checkstyle.xml</configLocation>
-              <includeTestSourceDirectory>true</includeTestSourceDirectory>
-          </configuration>
-        </plugin>
-
-        <plugin> <!-- JUnit report -->
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-surefire-report-plugin</artifactId>
-          <version>2.19.1</version>
-        </plugin>
-
-        <plugin>
-          <groupId>org.codehaus.mojo</groupId>
-          <artifactId>cobertura-maven-plugin</artifactId>
-          <version>2.7</version>
-        </plugin>
-
-        <!-- PMD configuration for JUnit tests -->
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-pmd-plugin</artifactId>
-          <version>3.6</version>
-          <configuration>
-              <skipEmptyReport>false</skipEmptyReport>
-              <includeTests>true</includeTests>
-              <rulesets>
-                  <ruleset>pmd-rules.xml</ruleset>
-              </rulesets>
-          </configuration>
-        </plugin>
-
-        <plugin>
-          <groupId>org.codehaus.mojo</groupId>
-          <artifactId>findbugs-maven-plugin</artifactId>
-          <version>3.0.3</version>
-          <configuration>
-            <xmlOutput>true</xmlOutput>
-            <includeTests>true</includeTests>
-          </configuration>
-        </plugin>
-      </plugins>
-    </reporting>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>nl.tudelft.pixelperfect</groupId>
+	<artifactId>pixelperfect</artifactId>
+	<version>0.4.0</version>
+	<name>PixelPerfect</name>
+	<description>Software project for the Computer Games Contextproject 2015-2016 (TI2806) at Delft University of Technology by group PixelPerfect.</description>
+
+	<properties>
+		<jme3.version>3.1.0-beta1</jme3.version>
+	</properties>
+
+	<repositories>
+		<repository>
+			<id>local</id>
+			<name>Local Repository</name>
+			<url>file://${basedir}/library</url>
```

Comment:
It's a shame a lot of dependencies had to be included locally in order to make this work. We will need to find a solution to host (at least the majority of) the dependencies online.